fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -921,6 +921,27 @@ private void analyzeAddPartition(OlapTable olapTable, List<PartitionDesc> partit
         }
     }
 
+    private int calAvgBucketNumOfLastFivePartitions(OlapTable olapTable) {
```

Comment:
Make 5 as a function parameter, only small changes needed when we want to change the strategy.